ReactTask/src/components/MainPatientsAppointment/MainPatientsAppointment.js

```diff
@@ -28,12 +38,22 @@ const MainPatientsAppointment = ({ switchContent }) => {
           <BtnCreateAppointment cliclChangeBox={switchContent} />
         </StylBoxDoctor>
       </StylBoxRowPadding>
-      {!!dataPatient.length && (
-        <CardPatientProfile
-          dataPatients={dataPatient}
-          imgPatientsArr={'imgArr'}
-          imgIconDescription={heartImgSvg}
-        />
+      {!!appointments.length && (
+        <StylBoxPatientsList>
+          {appointments.map(
+            ({ visitDate, reason, note, doctor, status }, index) => (
+              <CardPatientProfile
+                key={index}
+                status={status}
+                imgIconDescription={heartImgSvg}
+                visitDate={visitDate}
+                reason={reason}
+                note={note}
+                doctor={doctor}
```

Comment:
Why dont you import appointent into the component, and make Destructuring in inner component?